packages/ra-ui-materialui/src/input/PasswordInput.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import polyglotI18nProvider from 'ra-i18n-polyglot';
+import englishMessages from 'ra-language-english';
+
+import { AdminContext } from '../AdminContext';
+import { Create } from '../detail';
+import { SimpleForm } from '../form';
+import { FormInspector } from './common';
+import { PasswordInput } from './PasswordInput';
+
+export default { title: 'ra-ui-materialui/input/PasswordInput' };
+
+export const Basic = () => (
+    <Wrapper>
+        <PasswordInput source="password" />
+    </Wrapper>
+);
+
+export const FullWidth = () => (
+    <Wrapper>
+        <PasswordInput source="password" fullWidth />
+    </Wrapper>
+);
+
+export const Disabled = () => (
+    <Wrapper>
+        <PasswordInput source="password" defaultValue="password" disabled />
```

Comment:
**thought**: I'm wondering if this input should allow the user to click the "see password" button if it is disabled or read-only. An thought @slax57?

Reply:
I agree but this is really nitpicking and this PR is already long enough. Besides a disabled password input does not really make sense anyways. Let's keep things as they are for now.